### ISSUE TITLE
skip_verify in 0.7.1

### ISSUE THREAD

**Issue by ksator** [akarneliuk/pygnmi#69]
This is working 

- pygnmi 0.7.1
- token based auth
- no `username` and no `password`
- `skip_verify` instead of `override`
```
arista@devbox:~$ pip freeze | grep gnm
pygnmi==0.7.1
```
```
arista@devbox:~$ cat test2.py 
from pprint import pprint as pp
import json
from pygnmi.client import gNMIclient

with open("token.tok") as f:
    TOKEN = f.read().strip('\n')

host = ('192.168.0.5', '443')
with gNMIclient(target=host, token=TOKEN, skip_verify=False) as gc:
    result = gc.capabilities()

pp(result) 
arista@devbox:~$ 
```
but using `skip_verify=True` it doesnt work. 
It should be the opposite isnt it?

**Comment by bianca-mateoiu** [comment]
I second this, I have observed the same behaviour

**Comment by akarneliuk** [comment]
Hello folks,

thanks for reads up, can please post how your script looks like and the content of the certificate?

Best,
Anton

**Comment by akarneliuk** [comment]
Hey,

I've published the new pygnmi release `0.7.2`, which changes the logic for this `skip_verify` functionality a bit. Now it shall be working, but, please, pay attention to note in release notes.

Best,
Anton

**Comment by ksator** [comment]
it works using pygnmi 0.7.2 
many thanks @akarneliuk
